src/ucm/mmap/install.c

```diff
@@ -194,8 +218,37 @@ void ucm_fire_mmap_events(int events)
     ucm_fire_mmap_events_internal(events, &data);
 }
 
+static void ucm_mmap_event_report_missing(int expected, int actual,
+                                          int is_external)
+{
+    int events_count = 0;
+    int idx;
+    char *buf;
+    char *buf_p;
+    char *end_p;
+
+    UCS_STATIC_ASSERT(UCM_MMAP_REPORT_BUF_LEN <= UCS_ALLOCA_MAX_SIZE)
+
+    buf   = buf_p = ucs_alloca(UCM_MMAP_REPORT_BUF_LEN);
+    end_p = buf_p + UCM_MMAP_REPORT_BUF_LEN;
+
+    ucs_for_each_bit(idx,
+                     expected & ~actual & ((UCM_EVENT_VM_UNMAPPED << 1) - 1)) {
+        /* coverity[overrun-local] */
+        snprintf(buf_p, end_p - buf_p, "%s%s", (events_count ? ", " : ""),
```

Comment:
`(events_count > 0) ? ", " : ""`

Reply:
hmmm, 0 - is initial value, non-zero - all other states
but ok, fixed